omnibus/files/private-chef-cookbooks/private-chef/attributes/default.rb

```diff
@@ -39,11 +39,7 @@
 
 default['private_chef']['default_orgname'] = nil
 
-# Enable fips mode (openssl)
-# This requires the chef-server-fips package. If you do not have this,
-# your chef-server will probably not work. If you have to manually
-# change this, you're doing it wrong.
-default['private_chef']['fips_enabled'] = false
+default['private_chef']['fips_enabled'] = ChefConfig.fips?
```

Comment:
When you come back for cleanup tasks, it would be good to add a preflight check that we're on a box that supports fips if it has been enabled in the config.